Mathlib/Analysis/Convex/Between.lean

```diff
@@ -153,14 +153,14 @@ theorem Function.Injective.sbtw_map_iff {x y z : P} {f : P →ᵃ[R] P'} (hf : F
 @[simp]
 theorem AffineEquiv.wbtw_map_iff {x y z : P} (f : P ≃ᵃ[R] P') :
     Wbtw R (f x) (f y) (f z) ↔ Wbtw R x y z := by
-  refine Function.Injective.wbtw_map_iff (?_ : Function.Injective f.toAffineMap)
-  exact f.injective
+  have : Function.Injective f.toAffineMap := f.injective
+  apply Function.Injective.wbtw_map_iff this
```

Comment:
```suggestion
  exact this.wbtw_map_iff
```

Reply:
Careful: using dot notation works - using `exact` or `refine` is slow. `apply` is fast. I've switched to dot notation, but stayed with `apply`.

Reply:
Hmm, it was not noticeably slow when I tried it - certainly nowhere near what it was before. How much of a difference do you see?